bounties/level-2/stellar-accounts-as-a-service.md

```diff
@@ -0,0 +1,190 @@
+# Stellar accounts as a service
+
+## Award
+| Hunter | Reviewer | Type
+| :- | :- | :-
+| 2000 XLM | 800 XLM | Continuous
```

Comment:
Let's bump this to 2500 XLM and 1000 XLM. It's a really important bounty and a phenomenal educational resource. Let's ensure it's properly incentivized!

Reply:
I have changed this in a new commit.